Session/Meta/AppDelegate.swift

```diff
@@ -40,4 +40,20 @@ extension AppDelegate {
     @objc func stopClosedGroupPoller() {
         ClosedGroupPoller.shared.stop()
     }
+    
+    @objc func getAppModeOrSystemDefault() -> AppMode {
+        let userDefaults = UserDefaults.standard
+        
+        guard userDefaults.dictionaryRepresentation().keys.contains("appMode") else {
+            if #available(iOS 13.0, *) {
+                return UITraitCollection.current.userInterfaceStyle == .dark ? .dark : .light
+            } else {
+                return .light
+            }
+        }
+        
+        let mode = userDefaults.integer(forKey: "appMode")
```

Comment:
You could make this a bit nicer by doing `userDefaults[.appMode]`. See `SNUserDefaults` for how that works under the hood :)